subsys/instrumentation/Kconfig

```diff
@@ -89,16 +84,29 @@ config INSTRUMENTATION_TRIGGER_FUNCTION
 	default "main"
 	help
 	  Sets the trigger function. Instrumentation (tracing and profiling) is
-	  only turned on when the trigger function is called. The trigger
-	  function can be changed at runtime via the 'zaru' CLI tool.
+	  only turned on when the trigger function is called.
 
 config INSTRUMENTATION_STOPPER_FUNCTION
 	string "Default stopper function used to turn off instrumentation"
 	default "main"
 	help
 	  Sets the stopper function. Instrumentation (tracing and profiling) is
-	  only turned off when the trigger function returns. The stopper
-	  function can be changed at runtime via the 'zaru' CLI tool.
+	  only turned off when the stopper function returns.
+
+config INSTRUMENTATION_DYNAMIC_TRIGGER
```

Comment:
add `CONFIG_INSTRUMENTATION_DYNAMIC_TRIGGER` to the release notes (generally speaking, the new instrumentation subsytem seems to be completely missing from the release notes for now -- I'm happy if it's updated as a follow-up though)

Reply:
(doing that in https://github.com/zephyrproject-rtos/zephyr/pull/98069)